ci_framework/roles/ci_metallb/molecule/default/converge.yml

```diff
@@ -0,0 +1,198 @@
+                  - 172.18.0.80-172.18.0.90
+          controller:
+            default:
+              connection: ci-private-network
+              gw: 192.168.122.1
+              iface: eth1
+              ip: 192.168.122.11/24
+              mac: fa:16:3e:45:c1:b1
+              mtu: 1500
+            internal-api:
+              connection: ci-private-network-20
+              iface: eth1.20
+              ip: 172.17.0.4/24
+              mac: 52:54:00:30:d2:36
+              mtu: '1496'
+              parent_iface: eth1
+              vlan: 20
+          crc:
+            default:
+              connection: ci-private-network
+              gw: "{{ hostvars.crc.ansible_default_ipv4.gateway }}"
+              iface: "{{ hostvars.crc.ansible_default_ipv4.interface }}"
+              ip4: >-
+                {{
+                  hostvars.crc.ansible_default_ipv4.address +
+                    '/' + hostvars.crc.ansible_default_ipv4.prefix
+                }}
+              mtu: "{{ hostvars.crc.ansible_default_ipv4.mtu }}"
+            test-vlan-connection:
+              connection: test-network-20
+              iface: "{{ hostvars.crc.ansible_default_ipv4.interface }}.20"
+              ip4: 172.17.0.5/24
+              mtu: "{{ hostvars.crc.ansible_default_ipv4.mtu | string }}"
+              parent_iface: "{{ hostvars.crc.ansible_default_ipv4.interface }}"
+              vlan: 20
+
+    - name: Call the role under test
+      ansible.builtin.include_role:
+        name: "ci_metallb"
+
+    # Verify the run
+    - name: Fetch files stat results
+      ansible.builtin.stat:
+        path: >-
+          {{
+            [
+              ansible_user_dir,
+              'ci-framework-data',
+              'artifacts',
+              'manifests',
+              'metallb',
+              item
+            ] | ansible.builtin.path_join
+          }}
+      register: _ci_metallb_molecule_stat_out
+      loop:
+        - metallb-operatorgroup-olm.yaml
+        - metallb-subscription-olm.yaml
+        - ipaddresspools.yaml
+        - l2advertisements.yaml
+
+    - name: Assert that all expected files exist
+      ansible.builtin.assert:
+        that: >-
+          _ci_metallb_molecule_stat_out.results |
+          map(attribute="stat.exists") |
+          select("equalto", true) |
+          length == (_ci_metallb_molecule_stat_out.results | length)
+    - name: Fetch all files content
+      ansible.builtin.slurp:
+        path: "{{ item }}"
+      register: _ci_metallb_molecule_slurp_out
+      loop: >-
+        {{
+          _ci_metallb_molecule_stat_out.results |
+          map(attribute="stat.path")
+        }}
+    - name: Assert that all files contains proper YAML data
+      ansible.builtin.assert:
+        that: "item.content | b64decode | from_yaml_all | length > 0"
+      loop: "{{ _ci_metallb_molecule_slurp_out.results }}"
+      loop_control:
+        label: "{{ item.source }}"
+
+    - name: Wait for IPAddressPools to be ready
```

Comment:
Task name seems to not correspond with it's content.

Reply:
right, updated the task name to be more accurate